generator/gen_daal4py.py

```diff
@@ -942,11 +942,18 @@ def gen_daal4py(daalroot, outdir, version, warn_all=False, no_dist=False, no_str
     with open(jp(outdir, 'daal4py_cpp.cpp'), 'w') as f:
         f.write(cpp_cpp)
 
-
     with open(jp('src', 'gettree.pyx'), 'r') as f:
         pyx_gettree = f.read()
 
+    pyx_modelbuilder = ''
+    if 'algorithms::gbt::classification' in iface.namespace_dict and \
```

Comment:
I would transfer namespaces to dictionaries. See examples [here](https://github.com/IntelPython/daal4py/blob/master/generator/wrappers.py). This would make it easy to expand new builders without changing the generator.